packages/data-point/lib/reducer-path/factory.test.js

```diff
@@ -33,3 +33,116 @@ describe('reducer/reducer-path#create', () => {
     expect(reducer.asCollection).toBe(true)
   })
 })
+
+describe('ReducerPath#body', () => {
+  test('resolve "empty" jsonpath to entire value', () => {
+    const acc = {
+      value: 'test'
+    }
+    const result = factory.create('$').body(acc)
+    expect(result).toBe('test')
+  })
+
+  test('resolve "$." to entire value', () => {
+    const acc = {
+      value: 'test'
+    }
+    const result = factory.create('$.').body(acc)
+    expect(result).toBe('test')
+  })
```

Comment:
I realize this was in existence before, but there being two means of accessing the entire value is a little weird. Is some value to having both?

Reply:
I think this issue covers what you're talking about: https://github.com/ViacomInc/data-point/issues/52